src/posts/posts.service.ts

```diff
@@ -22,6 +22,10 @@ export class PostsService {
     return { id: createdPost.uuid };
   }
 
+  async deletePost(postid: string): Promise<void> {
+    await this.postRepository.deletePost(postid);
```

Comment:
I've a question here, shouldn't I return anything back to the client/user (who issued the delete request) when the post has successfully gets deleted ?

Reply:
What do you suggest we respond with to the user?

Reply:
Maybe a boolean that indicates if the post has been deleted successfully or not, because I think there should be some action that will be taken maybe like showing a message to the user that the post has been deleted or not.
What do you think?

Reply:
If an indication is what you're looking for, then this is already a sufficient indication. If you're suggesting that there would an action following the deletion process and dependent on some data coming from it, then please clarify as to what that action might be.
Also, this is a mere implementation of the contract. We have probably discussed this at some point in the past. So, if you are suggesting changes in the contract itself, then we should discuss this at a meeting or in the standup.

Reply:
Yes, its something that recently popped to my mind only.
I agree with you, but could you clarify this point to me more:
``then this is already a sufficient indication``?

Reply:
I can't prove that it is sufficient as this is probably one of many ways that are good enough to handle success response to a  `DELETE` request.
So I suggest we move this discussion to a meeting of some sort, so that we can share opinions on this matter as to what would be the best way to handle this successful deletion response.